### PR TITLE
added average goodness of fit values to high order tasks in ferengi t…

### PR DESCRIPTION
**Pull request by Mel23**
…o section 4

Need to discuss results, still not sure these stats are as convincing as visual inspection.

### CODE REVIEW

File: writeup/gz_hubble_data.tex [hunk 1]
```diff
@@ -475,7 +475,8 @@ \subsection{Challenges of debiasing questions beyond ``smooth or features''}
...
...dshift. This requirement is by default met by all galaxies when analyzing the first question, since it is asked of all users. This requirement is often \emph{not} met for questions beyond Task~01; on average 60.0\% of the galaxies do not have sufficient data to measure a correction, as compared to 2.0\% acheived for Task~01 (see Table~\ref{tbl:ferengi_corrections}). Of the remaining FERENGI galaxies which did meet this requirement, polynomials were fit to the data in each redshift-surface brightness bin, as was done for Task~01 (described above). The goodness-of-fit was evaluated in each bin, for each Task, using the value of the sum of the square residuals (SSR). The mean SSR for all Tasks was $\mu_{\rm SSR}$ = 0.46 $\pm$ 0.36, which was not considered sufficiently low in any Task for the fits to accurately model the data. An example of the debiasing technique applied to $p_{\rm bar}$ is shown in Section \ref{sec:ferengi_bar}; visual inspection of Figure~\ref{fig:p_vs_p_bar} confirms the available data in each bin is scattered to a degree at which no correlation may be confidentely obtained. For these reasons, debiased vote fractions are only offered for Task~01 (smooth/features).
```
Comment (willettk) [inline writeup/gz_hubble_data.tex]: confidentely -> confidently

**Comment by willettk** [comment]
My previous PR (#55) is going to overwrite parts of your paragraph. Can you pull in my new changes to master and resolve the conflicts in your files, then resubmit the PR?

**Comment by Mel23** [comment]
Conflicts taken care of, @willettk - I'm not sure if I need to close this request and start a new one or if this PR tracks it.

**Comment by willettk** [comment]
Any new changes you push to the branch on Github will update a PR.

**Pull request merged**